docs/serving/config-ha.md

```diff
@@ -29,6 +29,10 @@ $ kubectl -n knative-serving scale deployment <deployment-name> --replicas=2
 - Setting `--replicas=1` disables HA.
 - Passing `--disable-ha` to the controller process disables leader election.
 
+If you scale down the `autoscaler` component, you may get inaccurate autoscaling results for a subset of your Revisions for a peroid, which can
+be up to `stable-window` time. This is because when an `autoscaler` pods is terminating, the leadership of its owned Revisions will be gained by
```

Comment:
```suggestion
be up to `stable-window` time. This is because when an `autoscaler` pod is terminating, the ownership of the currently owned Revisions will be acquired by
```
or `pods are`